CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Contributing
+
+We'd love to have you join our group of contributors! Please e-mail your areas of interest and your availability to Alice (alice@code.org), and we’ll be happy to match you with a project. You can start setting up with these next steps.
+
+1. Anyone who would like to contribute to **[Code.org](https://github.com/code-dot-org/)** projects **must read and sign the [Contributor License Agreement](https://na2.docusign.net/Member/PowerFormSigning.aspx?PowerFormId=8eb90665-c9f7-4b06-81a5-11d12020f251)**. We can't accept pull requests from contributors who haven't yet signed the CLA.
+
+2. Get your local development build working by following our [README](README.md). Fork our repo and make sure to merge our staging branch into yours **WEEKLY** as we do update frequently.
+
+3. [Join our community development HipChat room](http://www.hipchat.com/gBebkHP6g) for help getting set up. We're happy to have you! If you want to make sure you get our attention, include an **@all** (everyone) or **@here** (everyone currently in the room) in your message.
+
+## Code Style
+
+Please make sure your code conforms to our [style guide](STYLEGUIDE.md).  We have implemented linting to help you with this:
+
+* To install our precommit hook that will lint the files you've changed, run `rake install:hooks`. You should only have to do this once.  We _highly_ recommend it!
+* To lint the entire project, run `rake lint` from the repository root.  This will check all Ruby, JavaScript and HAML.
+* To lint only part of the project, run one of these commands from the repository root:
+  * dashboard: See the [dashboard README](./dashboard#code-style)
+  * pegasus Ruby: `cd pegasus && rubocop`
+  * pegasus HAML: `haml-lint pegasus`
+  * apps: See the [apps README](./apps#style-guide)
+  * code-studio: See the [code-studio README](./code-studio#code-style)
+
+## Testing
+
+Please test your changes before submitting them to us!
+
+### Manually
+
+We support recent versions of Firefox, Chrome, IE9, iOS Safari and the Android browsers ([full list of supported browsers and versions](https://support.code.org/hc/en-us/articles/202591743)). Be sure to try your feature out in [IE9](docs/testing-ie9.md), iOS and Android if it's a risk. [BrowserStack live](http://www.browserstack.com) or [Sauce Labs manual](https://saucelabs.com/manual) let you run manual tests in these browsers remotely.
```

Comment:
Actually, it's funny to mention "recent versions of IE9" here.  I'll change that to "Internet Explorer" and add Edge (though I'm not sure we _officially_ support Edge yet)